changelog/db.changelog-1.8.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Steve Dickinson" id="1">
+    <tagDatabase tag="v1.8.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <modifyDataType columnName="name" newDataType="VARCHAR(30)" tableName="schemes" />
+    <modifyDataType columnName="suffix" newDataType="VARCHAR(20)" tableName="batchProperties" />
+  </changeSet>
+  <changeSet author="John Watson" id="3">
+    <insert tableName="schemes">
+      <column name="schemeId" value="3" />
+      <column name="name" value="Lump Sums" />
+    </insert>
+    <insert tableName="schemes">
+      <column name="schemeId" value="4" />
+      <column name="name" value="Vets Visits" />
+    </insert>
+    <insert tableName="schemes">
+      <column name="schemeId" value="5" />
+      <column name="name" value="LNR" />
+    </insert>
+    <insert tableName="sequences">
+      <column name="schemeId" value="3" />
+      <column name="nextAP" value="1" />
+      <column name="nextAR" value="1" />
+    </insert>
+    <insert tableName="sequences">
+      <column name="schemeId" value="4" />
+      <column name="nextAP" value="1" />
+      <column name="nextAR" value="1" />
+    </insert>
+    <insert tableName="sequences">
+      <column name="schemeId" value="5" />
+      <column name="nextAP" value="1" />
+      <column name="nextAR" value="1" />
+    </insert>
+    <insert tableName="batchProperties">
+      <column name="schemeId" value="3" />
+      <column name="prefix" value="FFCLS" />
+      <column name="suffix" value=" (LUMP)" />
+      <column name="source" value="SitiLUMP" />
+    </insert>
+    <insert tableName="batchProperties">
+      <column name="schemeId" value="4" />
+      <column name="prefix" value="FFCVV" />
+      <column name="suffix" value=" (Vets Visits)" />
```

Comment:
Should the README reflect these values -- I have no idea where these prefix and suffix would appear in the messages